### PR TITLE
add category-grid support

### LINKED ISSUE #70: columns (category-grid)
image
image

### PR DESCRIPTION
**Pull request by dkuntze**
Please always provide the [GitHub issue(s)](../issues) your PR is for, as well as test URLs where your change can be observed (before and after):

Fix #70 

Test URLs:
- Before: https://main--vonage--hlxsites.hlx.page/drafts/columns-category-grid
- After: https://70-columns-category-grid--vonage--hlxsites.hlx.page/drafts/columns-category-grid

### CODE REVIEW

File: blocks/columns/columns.css [hunk 1]
```diff
@@ -39,6 +39,18 @@ div.columns.border-bottom-gray {
   border-bottom: .1rem solid lightgray;
 }
 
+.category-grid p {
```
Comment (synox) [inline blocks/columns/columns.css:42]: I recommend to prefix each selector with `.columns `

File: blocks/columns/columns.css [hunk 2]
```diff
@@ -99,6 +111,20 @@ div.slim-promo a {
   text-decoration: underline;
 }
 
+.category-grid {
```
Comment (synox) [inline blocks/columns/columns.css:114]: this selector should be before `.category-grid p`

File: blocks/columns/columns.css [hunk 3]
```diff
@@ -99,6 +111,20 @@ div.slim-promo a {
...
+
+.category-grid h4 {
+  font-family: var(--fixed-font-family);
+  text-transform: uppercase;
+  font-size: 1.4rem;
+  font-weight: 600;
+  letter-spacing: .075rem;
+  line-height: 2rem;
+}
+
```
Comment (synox) [inline blocks/columns/columns.css:127]: I think it should be top-aligned, but it is not. You could try: 
.columns.category-grid > div {
  align-items: flex-start;
}

File: blocks/columns/columns.css [hunk 4]
```diff
@@ -39,6 +39,18 @@ div.columns.border-bottom-gray {
   border-bottom: .1rem solid lightgray;
 }
 
+.category-grid p {
+  font-size: 1.7rem;
```
Comment (synox) [inline blocks/columns/columns.css:43]: can we use one of the global font sizes? (ie. `--body-font-size-s`)

**Review by synox — CHANGES_REQUESTED** [review]
looks good, some minor changes

File: blocks/columns/columns.css (hunk 3 above)
Comment (dkuntze) [reply blocks/columns/columns.css]: I didn't see this issue, but I added it anyway.

Comment (synox) [reply blocks/columns/columns.css]: I still see it 
Screenshot 2023-06-21 at 14 40 55

Comment (dkuntze) [reply blocks/columns/columns.css]: you may have jumped the gun and previewed before I pushed up the change:

image

**Review by synox — APPROVED** [review]
👏

**Pull request merged**